istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -76,6 +76,14 @@ data:
     enableSdsTokenMount: true
     {{- end }}
 
+    {{- if .Values.useMCP }}
+    config_sources:
+    {{- if .Values.global.controlPlaneSecurityEnabled }}
+    - address: localhost:15019
+    {{- else }}
+    - address: istio-galley
```

Comment:
Please check the envoy file used for 'controlPlaneSecurityEnabled' - the address should have the suffix, since galley may not run in the same namespace (or cluster) with pilot.

Reply:
Technically it's not a regression - previous code also had it, so not blocking.